hoomd/md/integrate.py

```diff
@@ -121,25 +126,59 @@ class Integrator(_DynamicIntegrator):
     - `hoomd.md.methods.NPT`
     - `hoomd.md.methods.nph`
 
-    There can only be one integration mode active at a time. If there are more
-    than one ``integrate.mode_*`` commands in a hoomd script, only the most
-    recent before a given ```hoomd.run``` will take effect.
+    The classes of following modules can be used as elements in `forces`
+
+    - `hoomd.md.angle`
+    - `hoomd.md.bond`
+    - `hoomd.md.charge`
+    - `hoomd.md.dihedral`
+    - `hoomd.md.external`
+    - `hoomd.md.force`
+    - `hoomd.md.improper`
+    - `hoomd.md.pair`
+    - `hoomd.md.wall`
+    - `hoomd.md.special_pair`
+
+    The classes of the following module can be used as elements in `constraints`
+
+    - `hoomd.md.constrain`
 
     Examples::
+    
+        nlist = hoomd.md.nlist.Cell()
+        lj = hoomd.md.pair.LJ(nlist=nlist)
+        lj.params.default = dict(epsilon=1.0, sigma=1.0)
+        lj.r_cut[('A', 'A')] = 2**(1/6)
+        nve = hoomd.md.methods.NVE(filter=hoomd.filter.All())
+        integrator = hoomd.md.Integrator(dt=0.001, methods=[nve], forces=[lj])
+        sim.operations.integrator = integrator
+
+
+    Attributes:
```

Comment:
I would like to see the same documentation for the attributes as for the arguments here.

Reply:
@b-butler Same documentation means all the data type or descriptions as arguments? Descriptions on arguments have extra information of default value, comparing to Attributes.

Reply:
The sections should be mostly the same. Some the the exceptions are attributes that are not constructor arguments, type information where attributes have explicit types and arguments may be duck typed, and as you mention defaults.

Reply:
@joaander right, do you think I need to fix the data types on the attributes? `methods`, `forces`, `constraints` are returning their values with `list` type. and `dt` and `aniso` are returned into `float` and `str` type each.